src/Test/Utilities/Portable/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -671,6 +675,49 @@ string getRegionId(ControlFlowRegion region)
             {
                 return $"R{regionMap[region]}{idSuffix}";
             }
+
+            string getOperationTree(IOperation operation)
+            {
+                var walker = new OperationTreeSerializer(graph, currentRegion, idSuffix, anonymousFunctionsMap, compilation, operation, initialIndent: 8 + indent);
+                walker.Visit(operation);
+                return walker.Builder.ToString();
+            }
+        }
+
+        private class OperationTreeSerializer : OperationTreeVerifier
+        {
+            private readonly ControlFlowGraph _graph;
+            private readonly ControlFlowRegion _region;
+            private readonly string _idSuffix;
+            private readonly Dictionary<IFlowAnonymousFunctionOperation, ControlFlowGraph> _anonymousFunctionsMap;
+
+            public OperationTreeSerializer(ControlFlowGraph graph, ControlFlowRegion region, string idSuffix, 
+                                           Dictionary<IFlowAnonymousFunctionOperation, ControlFlowGraph> anonymousFunctionsMap,
+                                           Compilation compilation, IOperation root, int initialIndent) :
+                base(compilation, root, initialIndent)
+            {
+                _graph = graph;
+                _region = region;
+                _idSuffix = idSuffix;
+                _anonymousFunctionsMap = anonymousFunctionsMap;
+            }
+
+            public System.Text.StringBuilder Builder => _builder;
```

Comment:
Nit: Consider removing this property and replacing it either with a `ToString()` implementation or a `string GetOperationTree(...)` implementation